run_tests.py

```diff
@@ -0,0 +1,103 @@
+'''
+This script sets up role assumption for the MongoDB Module
+'''
+
+import datetime
+import json
+import os
+import subprocess
+import logging
+import click
+import sys
+
+# Set up logging, formatting and log level
+logging.basicConfig(
+    format='%(asctime)s [%(levelname)s] %(message)s', level=logging.INFO)
+
+
+@click.group()
+def cli():
+    pass
+
+
+@cli.command("test")
+@click.option('--skip-role-assumption', '-s', is_flag=True, help='Skip role assumption. Default: False')
+@click.option('--dir', type=click.Path(exists=True), default='test', help='The directory containing the tests. Default: test')
```

Comment:
Is this option needed since we will always only have tests in the test directory?

Reply:
@JosueMolinaMorales

Reply:
I guess not. It does default to test though